packages/web/components/Technology/Details/Tabs/FAQ/styles.js

```diff
@@ -0,0 +1,92 @@
+import styled, { css } from 'styled-components';
+
+export const Form = styled.form`
+	${({ theme: { screens } }) => css`
+		display: flex;
+		align-items: center;
+
+		> div {
+			width: 640px;
+			max-width: 60%;
+		}
+
+		@media (max-width: ${screens.medium}px) {
+			max-width: 100%;
+			flex-direction: column;
+			align-items: flex-start;
+
+			> div {
+				width: 100%;
+				max-width: 100%;
```

Comment:
Is it necessary, since they are both 100%?

Reply:
I do, because of the desktop styles.